### PR TITLE
Custom fields: multiple selection fields

### PR DESCRIPTION
**Pull request by WouterSioen**

### CODE REVIEW

**Review by mrtus — APPROVED** [review]

**Review by jenssegers — APPROVED** [review]

File: src/datastructures.apib [hunk 1]
```diff
@@ -66,3 +66,21 @@
         + end_of_month - End of the month of Xth day after the invoice date
         + after_invoice_date - X days after invoice date
 + days (number, optional) - Modifier "X" for the above statements. Not required when type is 'cash'.
+
+## CustomField (object)
++ definition (object)
+    + type: `customFieldDefinition` (string)
+    + id: `bf6765de-56eb-40ec-ad14-9096c5dc5fe1` (string)
++ One Of
```
Comment (jenssegers) [inline src/datastructures.apib:74]: Indentation issue?

Comment (WouterSioen) [reply src/datastructures.apib:74]: Why would there be an indentation issue? 🤔

Comment (jenssegers) [reply src/datastructures.apib:74]: Sorry, my bad!

Comment (WouterSioen) [reply src/datastructures.apib:74]: confused with the double `+` signs? 😄 It confused me too for a second

 :ok: by @WouterSioen on 30/03/2018 12:07:24

**Pull request merged**